api/overall_status.go

```diff
@@ -83,7 +83,7 @@ func (h *overallStatusHandler) ServeHTTP(w http.ResponseWriter, r *http.Request)
 			}
 		}
 
-		tasks, err := h.ctrl.Tasks(ctx)
+		tasks := h.ctrl.Tasks(ctx)
 		if err != nil {
```

Comment:
Is this error check still providing value, since the error assignment was removed from line 86?

Reply:
Thank you for catching